src/components/diagrams/singleLineDiagram/single-line-diagram-content.js

```diff
@@ -132,6 +144,28 @@ function SingleLineDiagramContent(props) {
         display: null,
     });
 
+    const [busMenu, setBusMenu] = useState({
+        position: [-1, -1],
+        busId: null,
+        //equipmentType: null,
```

Comment:
why commented code ?

Reply:
It's not needed anymore, removing it from powsybl-diagram-viewer as well
I thought I would need it to distinguish bus from busBarSection, but they are actually the same here